dockerfiles/buildall.sh

```diff
@@ -18,6 +18,11 @@ if [[ -n ${INPUT_USERNAME} ]] ; then
   PUSH=1
 fi
 
+if [[ -n ${INPUT_BUILDTYPE} ]] ; then
+  buildtype="--build-arg BUILD_TYPE=$INPUT_BUILDTYPE"
+  suffix=-$INPUT_BUILDTYPE
```

Comment:
The `buildtype` above is not "required", is the intention that blank is "Release"?  If so then the suffix will be different between undefined and Release?

Reply:
I want no suffix on the default (which is actually RelWithDebInfo), and a suffix only if a build type is explicitly specified.

Reply:
Ok by design,